### PR TITLE
Rewrote Redstone event using new ScriptEvent format.

### PR DESCRIPTION
**Pull request by Talamar1**
Converting Redstone event, yes mcmonkey, it appears that something stripped out the newline at the end of the file.

### CODE REVIEW

File: src/main/java/net/aufdemrand/denizen/events/scriptevents/RedstoneScriptEvent.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    // @Warning This event fires very very rapidly!
+    //
+    // @Triggers when a redstone wire is recalculated.
+    // @Context
+    // <context.location> returns the location of the block.
+    // <context.old_current> returns what the redstone power level was.
+    // <context.new_current> returns what the redstone power level is becoming.
+    //
+    // @Determine
+    // Element (Integer) set the current value to a specific value.
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/events/scriptevents/RedstoneScriptEvent.java]: Element(Number)

File: src/main/java/net/aufdemrand/denizen/events/scriptevents/RedstoneScriptEvent.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+import java.util.HashMap;
+
+public class RedstoneScriptEvent extends ScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // redstone recalculated
+    //
+    // @Cancellable true
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/events/scriptevents/RedstoneScriptEvent.java]: Remove this

Comment (Talamar1) [inline src/main/java/net/aufdemrand/denizen/events/scriptevents/RedstoneScriptEvent.java]: Remove the whole event?

Comment (Talamar1) [inline src/main/java/net/aufdemrand/denizen/events/scriptevents/RedstoneScriptEvent.java]: Removed:
   // @Cancellable true

File: src/main/java/net/aufdemrand/denizen/events/scriptevents/ItemMoveScriptEvent.java [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    // @Triggers when an entity or block moves an item from one inventory to another. (Hopper-style movement, not player-induced movement).
+    //
+    // @Context
+    // <context.origin> returns the origin dInventory.
+    // <context.destination> returns the destination dInventory.
+    // <context.initiator> returns the dInventory that initiatied the item's transfer.
+    // <context.item> returns the dItem that was moved.
+    //
+    // @Determine
+    // "CANCELLED" to stop the item from being moved.
```
Comment (mcmonkey4eva) [inline src/main/java/net/aufdemrand/denizen/events/scriptevents/ItemMoveScriptEvent.java]: Remove this, `@Cancellable` covers it

Comment (Talamar1) [inline src/main/java/net/aufdemrand/denizen/events/scriptevents/ItemMoveScriptEvent.java]: Done.

**Pull request closed without merging**